### PR TITLE
Chore: Update OWNERS and OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by tsmetana**
Changing the component as per the other drivers and adding Maxim in storage maintainers.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,3 +1,3 @@
 approvers:
 - openshift-storage-maintainers
-component: "Storage"
\ No newline at end of file
+component: "Storage / Kubernetes External Components"
```
Comment (bertinatto) [inline OWNERS:3]: In Azure Disk CSI Driver we use a `subcomponent` field: https://github.com/openshift/azure-disk-csi-driver/blob/master/OWNERS

Which way is the correct one?

Comment (tsmetana) [reply OWNERS:3]: This one. I have asked about it. :) We no longer have subcomponents in our bug tracking tools...

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**